src/planscape/forsys/scenario_sets.R

```diff
@@ -2,21 +2,24 @@ library(dplyr)
 library(forsys)
 library(sf)
 
-scenario_sets <- function(input_stand_data, priorities) {
-  input_stand_data
+scenario_sets <- function(input_stand_data, priorities, stand_id_field,
+                          proj_id_field, stand_area_field, stand_cost_field) {
+
+  scenario_output_fields <- append(priorities, c(stand_area_field, stand_cost_field))
 
   stand_dat <- test_forest %>% st_drop_geometry()
   run_outputs <- forsys::run(
```

Comment:
While we're waiting for a `print = FALSE` parameter in ForSys, you could try to disable output via something like this
```
print_dump <- capture.output(
  output <- suppressWarnings(
    forsys::run(
      ... <args here>
    )
  )
)
```

Reply:
cool!

I tried a few permutations of this but forsys was still printing to console.

```
suppressWarnings(
  run_outputs <- forsys::run(
    ...
  )
)
```

```
print_dump <- capture.output(
  output <- suppressWarnings(
    run_outputs <- forsys::run(
      ...
    )
  )
)
```

Reply:
(is this something you might want to look into? if so, and if you want help setting up your laptop environment to bring up your own django backend, we can plan a co-working session at the office sometime ...)

Reply:
fixed.
@JanLauGe suggested using "suppressMessages".